### PR TITLE
update user documentation for testing odbc driver connection on windows

### PR DESCRIPTION
**Pull request by rupal-bq**
*Issue #721*

*Description of changes:*
- added user documentation for testing connection using ODBC Data Source Administrator on windows

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sql-odbc/docs/user/windows_configure_dsn.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Configuring a DSN on Windows
+
+## Test Connection
+
+1. Open ODBC Data Source Administrator.
+
+ <img src="img/win_odbc_data_sources.png" width="30%">
+
+2. Click on `System DSN`. Select `ODFE SQL ODBC DSN` and click on `Configure`.
```
Comment (jordanw-bq) [inline sql-odbc/docs/user/windows_configure_dsn.md:9]: Make it clear that `System DSN` is a tab in the UI.
```suggestion
2. Click on the `System DSN` tab. Select `ODFE SQL ODBC DSN` and click on `Configure`.
```

File: sql-odbc/docs/user/windows_configure_dsn.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Configuring a DSN on Windows
+
+## Test Connection
```
Comment (jordanw-bq) [inline sql-odbc/docs/user/windows_configure_dsn.md:3]: Clarity that this section is for the default DSN installed with the Driver
```suggestion
## Test Default Connection
```

File: sql-odbc/docs/user/windows_configure_dsn.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ <img src="img/win_configure_dsn_values.png" width="80%">
+
+5. Click on `Test` to verify connectivity. You will get a message as `Connection successful`.
+
+ <img src="img/win_test_dsn.png" width="20%">
+
+6. Click on `OK` to save DSN values.
+
+
+ ## Add User DSN
```
Comment (jordanw-bq) [inline sql-odbc/docs/user/windows_configure_dsn.md:34]: Align header markers with margin (`##`) 
```suggestion
## Add User DSN
```

File: sql-odbc/docs/user/windows_configure_dsn.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+ The DSN configuration options can be set as shown below
+
+ <img src="img/win_configure_dsn_values.png" width="80%">
+
+5. Click on `Test` to verify connectivity. You will get a message as `Connection successful`.
+
+ <img src="img/win_test_dsn.png" width="20%">
+
+6. Click on `OK` to save DSN values.
+
```
[the remark this answers is not in the document]
Comment (rayl-bitquill) [reply sql-odbc/docs/user/windows_configure_dsn.md:32]: Let's create a low priority ticket to track enhancing this as well. Would prefer eventually that we auto-detect that encryption is enabled and take the appropriate action to make the user experience better. Not a high priority just want to track this.

[the remark this answers is not in the document]
Comment (rupal-bq) [reply sql-odbc/docs/user/windows_configure_dsn.md:34]: Added a note and created issue #726 to track this

**Review by rayl-bitquill — APPROVED** [review]

**Pull request merged**